CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,113 @@
+﻿This document was copied from its home at https://publiclab.org/conduct on October 12, 2017. See that page if you would like to submit your concerns in a safe, completely anonymous way, and to learn more about this document. 
```

Comment:
Let's update the date to current.

Reply:
@gauravano would you like me to do that? 

Sorry - I thought about doing that but decided to stick to the instructions in case that wasn't what you wanted.

Reply:
Yes, it wasn't mentioned in the instructions. I hope, it's fine with you?

Reply:
Hi - yes sure, that is fine. I have changed the date and pushed.

Reply:
Great!